src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -198,17 +197,39 @@ internal static void UpdateBackground(this EditText platformView, IView view)
 				mauiDrawable.Dispose();
 			}
 
-			// Android will reset the padding when setting a Background drawable
+			if (platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
+			// Android will reset the padding when setting a Background drawable	
 			// So we need to reapply the padding after
 			var padLeft = platformView.PaddingLeft;
 			var padTop = platformView.PaddingTop;
 			var padRight = platformView.PaddingRight;
 			var padBottom = platformView.PaddingBottom;
 
-			var previousDrawable = platformView.Background;
-			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
-			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, previousDrawable! });
-			platformView.Background = layer;
+			var paint = view.Background;
+
+			Drawable? defaultBackgroundDrawable = ContextCompat.GetDrawable(platformView.Context, Resource.Drawable.abc_edit_text_material);
+
+			var previousDrawable = defaultBackgroundDrawable ?? platformView.Background;
```

Comment:
Based on our conversation [here](https://github.com/dotnet/maui/pull/12227/files#r1131122599) it seems like we can just get rid of `previousDrawable` yea? Just use `defaultBackgroundDrawable` everywhere instead?

Reply:
Could simplify the code and normally will work in the same way. But if for any reason trying to get `Resource.Drawable.abc_edit_text_material` fails and we get a null Drawable, all the entries will not show the bottom line. With `previousDrawable` we use the default Drawable used by the AppCompatEditText.